src/Tests.IntegrationTests/ClassifyMessageTests.cs

```diff
@@ -39,7 +39,7 @@ public async Task EnvironmentIsConfigured()
             Assert.Equal(participants[1].InstitutionId, institutions.Single().Id);
         }
 
-        [Fact(Timeout = 120000)]
+        [Fact(Timeout = 240000)]
```

Comment:
Just adding a comment to hold this until we are 100% sure this is going to fix the intermittent test issues.

Reply:
Apropos of nothing:

An old skool programming trick to make these more readable is to leave the value like this:

Timeout = 4 * 60 * 1000

It's easier for simple minds (mine) to read and change later.

Reply:
Thats a fair suggestion, I'll add it

Reply:
I've tested the integration test against this increased timeout 4 times via CI and 5 times locally and have not hit the intermittent issues I was facing before the change. Therefore, I'm satisfied that this change is appropriate to make. As to why it occasionally takes so long to complete can be addressed separately.